cloudformation/main.yaml

```diff
@@ -14,11 +46,35 @@ Resources:
           - MailmanRegisterLambdaInvocationRole
           - Arn
 
-  # IAM permissions for lambda
+  # IAM permissions for lambdas
+  MailmanLambdaInvocationRole:
```

Comment:
Wouldn't the main lambda need dynamodb permissions?

Reply:
I was planning to add that in a separate diff once we update the main lambda to fetch from DB.